public/assets/js/blogPosts.js

```diff
@@ -0,0 +1,135 @@
+const createForm = $("#create-form");
+const updateForm = $("#update-form");
+const deletePost = $("#deletePost");
+const viewPost = $("#homePosts");
+const postComment = $("#postComment");
+
+const validate = (title, content) => {
+  // See if values contain text
+  newTitle = title.replace(/\s/g, "");
```

Comment:
Please be careful with code quality - i feel i've mentioned this a lot on previous homeworks, but it is bad code to declare variables without `let` or `const`